frontend/src/components/dashboard/AddonData.tsx

```diff
@@ -0,0 +1,40 @@
+import { getPlan, isPremiumAvailableInCountry } from "../../functions/getPlan";
+import { AliasData } from "../../hooks/api/aliases";
+import { ProfileData } from "../../hooks/api/profile";
+import { RuntimeData } from "../../hooks/api/runtimeData";
+
+export type Props = {
+  profile: ProfileData;
+  runtimeData: RuntimeData;
+  aliases: AliasData[];
+  totalForwardedEmails: number;
+  totalBlockedEmails: number;
+};
+
+export const AddonData = (props: Props) => {
+  return (
+    <firefox-private-relay-addon-data
+      // #profile-main is used by the add-on to look up the API token.
+      // TODO: Make it look for this custom element instead.
```

Comment:
thought: we should open an issue for this todo - great React practice.

Reply:
Good one! Otherwise it'll never happen :) Created https://github.com/mozilla/fx-private-relay-add-on/issues/333.